src/features/areaAssignments/l10n/messageIds.ts

```diff
@@ -15,12 +15,12 @@ export default makeMessages('feat.areaAssignments', {
       confirm: m(
         'Do you want to delete all unsaved changes and go back to saved instructions?'
       ),
-      editorPlaceholder: m('Add instructions for your callers'),
+      editorPlaceholder: m('Add instructions for your activists'),
       revertLink: m('Revert to saved version?'),
       saveButton: m('Save'),
       savedMessage: m('Everything is up to date!'),
       savingButton: m('Saving...'),
-      title: m('Caller instructions'),
+      title: m('Activist instructions'),
```

Comment:
There are two `title` keys, this one on line 23 and one on line 26. The only one that is being used is the one on line 26. So I think this message can be deleted.

Reply:
Removed line 23 !